### ISSUE TITLE
The token for allowing media autoplay with sound should be called "autoplay", not "media"

### ISSUE THREAD

**Issue by othermaciej** [WICG/gesture-delegation#13]
iframes are allowed to have media by default, they don't need special permission. The permission actually being delegated is to autoplay media.

**Comment by mounirlamouri** [comment]
The feature is not about delegating a permission but a user gesture. Here, the goal is to delegate the user gesture for media-related features. We could name it "autoplay" but for consistency with other keywords, it would be a bit inconsistent as "vibration" is the name of the feature that is allowed to use the gesture and "autoplay" isn't really a feature but the playback of media without a user gesture.

**Comment by othermaciej** [comment]
It's not actually delegating a user gesture. It's delegating a "sticky user gesture", i.e. the permission to do something that comes from having performed a user gesture at some point in the past. The attribute name is also assuming that permission specifically comes from a sticky user gesture, which it might not in other browsers.

**Comment by annevk** [comment]
If we called it "autoplay" it would make more sense in the context of Feature Policy allowing this to be reconciled perhaps.

**Comment by annevk** [comment]
I guess #4 was about that too.

cc @jyasskin

**Comment by othermaciej** [comment]
Yeah, now that I've read over Feature Policy, I think an "autoplay" Feature Policy token makes sense as a solution for this, and will fit with both Chrome and Safari autoplay restriction policies.

**Comment by ghost** [comment]
There is a proposal to add this to feature policy here: https://github.com/WICG/feature-policy/pull/93
